pact-broker/templates/osba-db-instance.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
   name: {{ .Release.Name }}-postgresql-instance
 spec:
   clusterServiceClassExternalName: azure-postgresql-9-6
-  clusterServicePlanExternalName: basic
+  clusterServicePlanExternalName: {{ required "postgresqlPlan " .Values.postgresqlPlan }}
```

Comment:
is the required needed? you are setting a default

Reply:
better safe than sorry :-)